coreapps-web-2.2/src/test/java/org/openmrs/module/coreapps/diagnosisCompatibility/DiagnosisServiceCompatibility2_2Test.java

```diff
@@ -0,0 +1,37 @@
+package org.openmrs.module.coreapps.diagnosisCompatibility;
+
+
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.Diagnosis;
+import org.openmrs.Encounter;
+import org.openmrs.api.EncounterService;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+public class DiagnosisServiceCompatibility2_2Test  extends BaseModuleContextSensitiveTest {
+	
+	@Autowired
+	@Qualifier("encounterService")
+	EncounterService encounterService;
```

Comment:
It doesn't matter much in test classes, but it's a bad habit to have [package scoped](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html) members by default.
In short, make this `private`.